.travis.yml

```diff
@@ -36,66 +34,35 @@ jobs:
         - pip3.7 install pyOpenSSL
         - pip3.7 install coveralls
       script:
-        # we want to fail fast on tox errors without having to `docker build` first
+        # ignores test_pythonpackage.py since it runs for too long
         - tox -- tests/ --ignore tests/test_pythonpackage.py
-        # (we ignore test_pythonpackage.py since these run way too long!!
-        #  test_pythonpackage_basic.py will still be run.)
       name: "Tox Pep8"
       env: TOXENV=pep8
-    - <<: *linting
+    - <<: *unittests
       name: "Tox Python 2"
       env: TOXENV=py27
-    - <<: *linting
+    - <<: *unittests
       name: "Tox Python 3 & Coverage"
       env: TOXENV=py3
       after_success:
         - coveralls
-
-    - &testing
-      stage: test
-      before_script:
-        # build docker image
-        - docker build --tag=p4a --file Dockerfile.py3 .
-        # Run a background process to make sure that travis will not kill our tests in
-        # case that the travis log doesn't produce any output for more than 10 minutes
-        - while sleep 540; do echo "==== Still running (travis, don't kill me) ===="; done &
```

Comment:
I suspect that we still may need this lines, not for the basic tests we make but for `rebuild_updated_recipes` whenever we touch some of the most time demanding recipes (`icu`, `boost+libtorrent`...). Anyway, we can leave it out, for now, and reintroduce it later in case that we need it...if not...better for us...less code to maintain and more readable.

Reply:
Good point for the demanding recipes. Yes I would try without for now as it's more readable, but I can definitely try with some of theses recipes on my fork to see if it builds for too long without providing any output